stable/traefik/values.yaml

```diff
@@ -58,6 +60,14 @@ service:
   #   key: value
   # labels:
   #   key: value
+  ## Further config for service of type NodePort
+  ## Configure ports in allowable range (eg. 30000 - 32767 on minikube)
+  ## nodePorts:
+  ##   http: 30080
+  ##   https: 30443
+  nodePorts:
+    http: ""
+    https: ""
```

Comment:
This is a bit awkward because the default values are empty strings, but any other value you enter here should be an int, right?

If you're going to do this, I'd suggest documenting it better.